cli/tests/integrations/test_service.py

```diff
@@ -16,29 +20,52 @@ def zk_znode(request):
     return request
 
 
+def setup_module(module):
+    package_install('chronos', True)
+
+
+def teardown_module(module):
```

Comment:
I think they solve different problems.  For the use case of "I want to do X when this module starts and Y when it stops", I think we should use setup/teardown